### ISSUE TITLE
Resident Evil 4, after some action hair will move slowly

### ISSUE THREAD

**Issue by MotokaJuu** [NSACloud/RE-Chain-Editor#3]
After Some CG or QTE hair's (cha000_20) motion'll become slow. but clothes (cha000_00, cha001_00, cha002_00) are good. 
Especially after Villiage bell ring CG and get bited by Big fish in lake

**Comment by NSACloud** [comment]
Does the problem happen on vanilla chain files that have only been imported and exported back out with no changes made?

**Comment by MotokaJuu** [comment]
no, it's new one. I make it for my mod. It's cha000_20.chain.53. It's only happen on it. although cha000_00.chain.53 and cha002_00.chain.53 are new, they just fine.
